test/StatsParent-test.js

```diff
@@ -0,0 +1,77 @@
+const chai = require('chai');
+const expect = chai.expect;
+
+import activityData from '../data/activity-test-data';
+import hydrationData from '../data/hydration-test-data';
+import sleepData from '../data/sleep-test-data';
+import userData from '../data/users-test-data';
+import allSleepData from '../data/sleep';
+
+import StatsParent from '../src/StatsParent';
+import Activity from '../src/Activity';
+import ActivityRepo from '../src/ActivityRepo';
+import Hydration from '../src/Hydration';
+import Sleep from '../src/Sleep';
+import SleepRepo from '../src/SleepRepo';
+import User from '../src/User';
+
+describe('Stats Parent Class', () => {
+  let statsParent;
+  let hydration;
+  let sleep;
+  let user;
+
+  beforeEach(() => {
+
+    statsParent = new StatsParent()
+    user = new User(userData[0])
+    // hydration = new Hydration(hydrationData, user.id)
+  });
+
+  it('should return the dates for the last week', () => {
+    expect(statsParent.returnWeek(1, hydrationData, user.id)).to.eql(
+      [
+        "2019/06/16",
+        "2019/06/17",
+        "2019/06/18",
+        "2019/06/19",
+        "2019/06/20",
+        "2019/06/21",
+        "2019/06/22"
+      ])
+    expect(statsParent.returnWeek(1, sleepData, user.id)).to.eql(
+      [
+        "2019/06/19",
+        "2019/06/20",
+        "2019/06/21",
+        "2019/06/22",
+        "2019/06/23",
+        "2019/06/24",
+        "2019/06/25",
+      ]);
+  });
+  it('should return the average data over all time for a single user', () => {
+    expect(statsParent.returnAvgData(sleepData, user.id, 'sleepQuality')).to.equal(2.53);
+    expect(statsParent.returnAvgData(sleepData, user.id,'hoursSlept')).to.equal(7.66);
+    expect(statsParent.returnAvgData(hydrationData, user.id,'numOunces')).to.equal(62.13);
+  });
+
+  it('should return data each day for week for a specific user', () => {
+    expect(statsParent.returnWeekOfSpecificData(2, allSleepData, user.id, 'hoursSlept')).to.eql([7.3, 5.1, 8.6, 10.5, 9.1, 6.5, 6.8]);
+  });
+
+  it('should return user data for a specific day', () => {
```

Comment:
The testing lesson feels like it happened ages ago so I'm not 100% sure this is a good case to create a new describe block, but I was thinking maybe it would be good to break these tests apart and have  it blocks for each data type (steps, hydration, sleep, and activity) so that anyone going through our tests know exactly what each test is for.  Let me know your thoughts on this idea.

Reply:
I'd agree with what Colin said, it keeps would keep what's being tested more immediately clear

Reply:
@spaceplesiosaur is this the section of the tests your question was referring to? I think it makes sense to test that the methods work expectedly for each kind of data they'll be using, but it would be good to make a nested describe block for the method and test each dataset in isolation (within their own `it` blocks).

Reply:
Sounds good, thanks @khalidwilliams !